Formula/gedit.rb

```diff
@@ -97,7 +96,7 @@ def post_install
       -I#{gtksourceview4.opt_include}/gtksourceview-4
       -I#{gtkx3.opt_include}/gtk-3.0
       -I#{harfbuzz.opt_include}/harfbuzz
-      -I#{include}/gedit-3.36
+      -I#{include}/gedit-3.38
```

Comment:
```suggestion
      -I#{include}/gedit-#{version.major_minor}
```

Reply:
Bad idea, Gedit actually rarely bumps this number. It's quite exceptional that it has been bumped twice in a row.